cmake/common.cmake

```diff
@@ -270,8 +270,21 @@ endif()
 find_file(CPPLINT NAMES cpplint cpplint.exe)
 if(CPPLINT)
     function(new_linter_target NAME SRC)
+        set(options)
```

Comment:
Are these 3 lines not redundant? Left over copy-paste from the docs for `cmake_parse_arguments()`?